yarn.lock

```diff
@@ -640,10 +682,10 @@ ansi-styles@^4.0.0, ansi-styles@^4.1.0:
   dependencies:
     color-convert "^2.0.1"
 
-ansi_up@^5.0.0:
-  version "5.0.0"
-  resolved "https://registry.yarnpkg.com/ansi_up/-/ansi_up-5.0.0.tgz#9bc3b087a59bd543c98dd235a9c8e0a842c454e9"
-  integrity sha512-RHw/w3Kb2U3k4XKfl8FXZW9ldxtTBbLNdKO0RboYeU4ReVwRP77M7b/OxiavMGZsBWcDxn/T0QiR+VtLf7mPYw==
+ansi_up@^4.0.4:
```

Comment:
`ansi_up` should be at least version 5.0.0, as in the `4.0.4` version there's a medium-severity vulnerability.

Reply:
> \>yarn audit
yarn audit v1.22.10
0 vulnerabilities found - Packages audited: 786

Yarn tells me everything is fine. Where did you find this?

Reply:
Dependabot reported an error previously. Let's see if that happens when we merge this PR...

Reply:
okay